docs/static/monitoring/monitoring-ea-serverless.asciidoc

```diff
@@ -0,0 +1,125 @@
+[[serverless-monitoring-with-elastic-agent]]
+=== Collect {ls} monitoring data for {serverless-full} dashboards
```

Comment:
Do we have a standard way to express serverless environments? I wonder if we could express it differently, something like:

Collect Logstash monitoring data for dashboards (Serverless)
Collect Logstash monitoring data for dashboards in a serverless environment